module/data/actor/templates/attributes.mjs

```diff
@@ -105,17 +105,17 @@ export default class AttributesFields {
    */
   static prepareMovement() {
     const statuses = this.parent.statuses;
-    const noMovement = new Set(["grappled", "paralyzed", "petrified", "restrained", "stunned", "unconscious"])
-      .intersection(statuses).size || (this.attributes.exhaustion >= 5);
-    const halfMovement = statuses.has("prone") || (this.attributes.exhaustion >= 2);
+    const noMovement = this.parent.hasConditionEffect("noMovement") || (this.attributes.exhaustion >= 5);
+    const halfMovement = this.parent.hasConditionEffect("halfMovement") || (this.attributes.exhaustion >= 2);
     const reduction = statuses.has("heavilyEncumbered")
       ? CONFIG.DND5E.encumbrance.speedReduction.heavilyEncumbered
       : statuses.has("encumbered") ? CONFIG.DND5E.encumbrance.speedReduction.encumbered : 0;
-    const crawl = statuses.has("prone") || statuses.has("exceedingCarryingCapacity");
+    const exceeding = statuses.has("exceedingCarryingCapacity");
+    const crawl = this.parent.hasConditionEffect("crawl") || exceeding;
```

Comment:
I'd vote to go back to the previous version, since that would allow customization of how `exceedingCarryingCapacity` behaves beyond simply enabling and disabling it in the settings.

Reply:
Not that I feel strongly about it, I simply didn't think that the subsystems of statuses and encumbrance should have overlapped as such.